content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -71,42 +113,80 @@ set(FILES
 )
 ```
 
+### Adding a temporary auto-component
+{{< note >}}
+There's currently an [issue](https://github.com/o3de/o3de/issues/4058) causing a build failure if multiplayer auto-components are enabled, but no auto-components are created. As a work-around, create a temporary auto-component.
+{{< /note >}}
```

Comment:
Let's reword this a bit the the probability that this note  ends up hanging around past it's useful life.

```suggestion
{{< note >}}
You might experience a build failure if multiplayer auto-components are enabled, but no auto-components are created. As a work-around, create a temporary auto-component. Refer to this [issue](https://github.com/o3de/o3de/issues/4058) for more information.
{{< /note >}}
```